### PR TITLE
RTEMIS-290: Modified the school fee detail entity & claim entity.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-290

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Modified the school claim entity to have dynamic academic year.
- Added access checks for school claim entity.
- Modified the user permissions.
- Added education level to the school fee details entity.

### CODE REVIEW

File: modules/rte_mis_reimbursement/src/Services/RteReimbursementHelper.php [hunk 1]
```diff
@@ -181,7 +183,51 @@ public function loadStudentData(?string $school_id = NULL, ?string $academic_yea
...
+        $government_tution_fees = 0;
+        $education_level = NULL;
+
+        if (isset($current_user_fee_details)) {
+          // $combination = $student_gender;
+          $gender_priorities = [
+            'boy' => ['boys', 'co-ed'],
+            'girl' => ['girls', 'co-ed'],
+            'transgender' => ['co-ed', 'boys', 'girls'],
+          ];
```
Comment (kunal-sin) [inline modules/rte_mis_reimbursement/src/Services/RteReimbursementHelper.php:196]: Can't we fetch this from the rte_mis_school configuration?

Comment (RohanSingh-inno) [reply modules/rte_mis_reimbursement/src/Services/RteReimbursementHelper.php:196]: These priorities are not stored in school config, we have defined this for this particular task only.

Comment (RohanSingh-inno) [reply modules/rte_mis_reimbursement/src/Services/RteReimbursementHelper.php:196]: @kunal-sin Should we store it in config and then use this?

Comment (kunal-sin) [reply modules/rte_mis_reimbursement/src/Services/RteReimbursementHelper.php:196]: `boys`, ``co-ed, ``girls this we can get from the config, right?

Comment (RohanSingh-inno) [reply modules/rte_mis_reimbursement/src/Services/RteReimbursementHelper.php:196]: Are we talking about the label here?

Comment (RohanSingh-inno) [reply modules/rte_mis_reimbursement/src/Services/RteReimbursementHelper.php]: done.

**Pull request merged**